services/src/test/resources/spring-test/cdr-client-container.xml

```diff
@@ -186,8 +184,4 @@
         class="org.springframework.web.multipart.commons.CommonsMultipartResolver">
         <property name="maxUploadSize" value="2000000000"/>
```

Comment:
Do we want this to be hard coded?

Reply:
Good point, I've created a ticket https://jira.lib.unc.edu/browse/BXC-3222

Reply:
See https://jira.lib.unc.edu/browse/BXC-3222